test/helpers/multisig.js

```diff
@@ -0,0 +1,48 @@
+const deployOrGetRootRegistry = require('./contracts').deployOrGetRootRegistry;
+
+const prepareMultiSigAndRoot = async config => {
+  const { network, artifacts, accounts } = config;
+  const admin0 = accounts[0];
+  const admin1 = accounts[1];
```

Comment:
use array destructuring

we should copy over the eslint config from the monorepo.